server/data/ERRORS.php

```diff
@@ -415,4 +419,5 @@ class ERRORS {
     const INVALID_SUPERVISED_USERS = 'INVALID_SUPERVISED_USERS';
     const INVALID_USER_SEARCH_OPTION = 'INVALID_USER_SEARCH_OPTION';
     const TICKET_CONTENT_CANNOT_BE_EDITED = 'TICKET_CONTENT_CANNOT_BE_EDITED';
+    const ERROR_PASSWORD = 'ERROR_PASSWORD';
```

Comment:
Let's use `INVALID_PASSWORD` instead, that already existed in backend, and replace all the usages of `ERROR_PASSWORD` in the frontend, to use `INVALID_PASSWORD` instead.